package/libyui-rest-api.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Aug 28 10:10:00 UTC 2020 - Joaquín Rivera <jeriveramoya@suse.com>
+
+- Serialize value for YDateField and YTimeField
```

Comment:
OBS complains in case  there is no bug reference in the change log (there are some other references being accepted). Could you please add `(bsc#1139747)`?